docs/src/pages/[platform]/connected-components/authenticator/advanced/useAuthenticator.angular.mdx

```diff
@@ -20,15 +20,16 @@ import { Component } from '@angular/core';
 import { Amplify } from 'aws-amplify';
 import { AuthenticatorService } from '@aws-amplify/ui-angular';
 
-import awsExports from './aws-exports';
+// import config from './aws-exports'; // Amplify Gen 1 config
+import config from './amplify_outputs.json'
 
 @Component({
   selector: 'app-root',
   templateUrl: 'app.component.html',
 })
 export class UseAuthenticatorComponent {
   constructor(public authenticator: AuthenticatorService) {
-    Amplify.configure(awsExports);
+    Amplify.configure(config);
```

Comment:
I've updated all the angular configs  instead of removing them because otherwise customers wouldn't know that they have to call Amplify.configure() in the constructor.